manticore/platforms/linux.py

```diff
@@ -1735,7 +1735,7 @@ class SLinux(Linux):
     :param list envp: environment variables
     :param tuple[str] symbolic_files: files to consider symbolic
     """
-    def __init__(self, programs, argv=None, envp=None, symbolic_files=()):
+    def __init__(self, programs, argv=None, envp=None, symbolic_files=[]):
```

Comment:
we should not use a list here because it's a mutable type. we should do the `None` thing like with argv/envp http://python-guide-pt-br.readthedocs.io/en/latest/writing/gotchas/#mutable-default-arguments